components/nimbus/ios/Nimbus/Nimbus.swift

```diff
@@ -48,9 +48,9 @@ private extension Nimbus {
 
 // Glean integration
 internal extension Nimbus {
-    func recordExposure(experimentId: String) {
+    func recordExposure(_ featureId: String) {
         let activeExperiments = getActiveExperiments()
-        if let experiment = activeExperiments.first(where: { $0.slug == experimentId }) {
+        if let experiment = activeExperiments.first(where: { $0.featureIds.contains(featureId) }) {
```

Comment:
It took me a minute to persuade myself that this worked.

My reasoning: 

1. `getActiveExperiments` gets a list of experiments that are enrolled.
2. Features are only enrolled in zero or one experiment at a time.
3. The first experiment we find where the `featureIds` contains our `featureId` will therefore be the only experiment enrolled involving this experiment.

For the purposes of this PR: let's add a comment persuading the reader that this works.

Moving forward: Adding Rust FFI surface area in the same PR should be fine — previously it wasn't. We should probably keep this, for the sake of shipping, but start the habit of treating the FFI as internals, not an API itself.

Reply:
I foresee where this entire function will be moved to Rust and can just be called across the FFI and recorded in Glean there (once I can get to it). So, rather than touching the Rust now and potentially making even more surface area that might need refactored when that happened, it seemed useful for me to try and keep this isolated in the foreign language bindings where the Glean integration is currently.

I agree definitely about adding a comment to clarify what exactly we are expecting from this line.